db/migrate/20150921183945_add_colunmns_admin_avatar_username_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddColunmnsAdminAvatarUsernameToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, default: false, null: false
+    add_column :users, :username, :string, null: false
+    add_column :users, :avatar, :string, null: false,
+               default: "http://www.besttats.com/pictures/tech-savy-back-tattoo.jpeg"
```

Comment:
Line is too long. [85/80]<br>Align the elements of a hash literal if they span more than one line.